src/test/java/com/wikia/webdriver/testcases/widgettests/AllTagsTests.java

```diff
@@ -41,8 +41,9 @@ public void prepareTest() {
     widgets.add(new VKWidgetPageObject(driver));
     widgets.add(new WeiboWidgetPageObject(driver));
 
-    // Uncomment after 22-sep
+    //TODO: Uncomment after 22-sep
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)